possible_topics.md

```diff
@@ -0,0 +1,51 @@
+# Possible topics for video content
+
+This document aims to collate possible ideas/outlines for video content
+related to onboarding (or otherwise encouraging) NumPy developers.
+
+## Workflow basics
+
+Start with the basic `git`/`github` workflow
+ - Fork, clone, set upstream
+
+Then on to setting up an environment
+ - There are a lot of options here (`conda`, `venv`, etc.) How to deal with the
+   various options?
+    * Personally, I like `venv` as it's Python standard library
+      and doesn't require any special knowledge of package managers
```

Comment:
Why not both? :) I'd be willing to contribute a video with conda instructions - it fits nicely into the workflow of many people already working in scientific computing.

Reply:
I think conda is unquestionably the way to go for windows (and probably mac) users, so from the perspective of casting a wide net I think starting with conda makes sense :+1: